suite-native/module-onboarding/src/screens/WelcomeScreen.tsx

```diff
@@ -91,12 +92,11 @@ export const WelcomeScreen = () => {
                 style={applyStyle(imageContainerStyle)}
             />
             <Screen backgroundColor="transparent">
-                <LinearGradient
+                <Box
```

Comment:
We agreed with @shenkys ditch the gradient and unify the backgrounds for all screens. However, the blur seems to be problematic since it may only be applied to an image and the image covers a bigger area. Solved by transparency instead since onboarding is to be reworked.

Reply:
I don't think so because `OnboardingScreen` has a header with stepper. 😕

Reply:
Can the header be optional and then reusable?

Reply:
OK, now I understand what you meant! I gave it a try and I guess it looks a bit better. 😎